infra/load-balancer.tf

```diff
@@ -0,0 +1,25 @@
+resource "helm_release" "load_balancer_controller" {
+  name       = "jovand-loadbalancer-controller"
+  repository = "https://aws.github.io/eks-charts"
+  chart      = "aws-load-balancer-controller"
+  namespace  = "default"
```

Comment:
Create separate namespace as on LB controller is supposed to be used for multiple namespaces